eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -193,15 +193,14 @@ function Set-ChangeLogContent {
 
   try
   {
-    $VersionsSorted = [AzureEngSemanticVersion]::SortVersionStrings($ChangeLogEntries.Keys)
+    $ChangeLogEntries = $ChangeLogEntries.Values | Sort-Object -Descending -Property ReleaseStatus, ReleaseVersion
```

Comment:
By sorting by `ReleaseStatus` (date or "(Unreleased)") first, it actually means any older entries without dates get sorted earlier in the final content. Probably worth a one-time fixup wherever necessary for customers' benefit.

Reply:
Yes I'm fine with having some older entries getting manually fixed up, we were really already depending on that.

With this change does "(Unreleased)" end up sorting to the top?

Reply:
Yep. Fortunately letters sort after numbers, so when we reverse the order it works out nicely. :)